### PR TITLE
User controller actions, edition of profile, review of profile, test

### LINKED ISSUE #37: User profile view

### LINKED ISSUE #38: User profile edition

### PR DESCRIPTION
**Pull request by KamilFilip**
closes #37 
closes #38

### CODE REVIEW

File: spec/controllers/user_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    it "checks if update changes parameters" do
+      put :update, params: {id: user.id, user: user_new_parameters}
+      user.reload
+      expect(user.name).to eql user_new_parameters[:name]
+      expect(user.surname).to eql user_new_parameters[:surname]
+      expect(user.city).to eql user_new_parameters[:city]
+    end
+
+    it "check if redirects to url" do
+      put :update, params: {id: user.id, user: user_new_parameters}
```
Comment (bpietraga) [inline spec/controllers/user_controller_spec.rb]: put space before and after {}

File: app/views/users/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.container
+  h3 Edit your profile
+  .grid.grid-pair
+    .grid-item class="3/4"
```
Comment (bpietraga) [inline app/views/users/edit.html.slim]: maybe 4/4?

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to root_url
+    else
+      render { action 'edit' }
+    end
+  end
+
+  private
+
```
Comment (bpietraga) [inline app/controllers/users_controller.rb]: unnecessary space

File: app/views/users/edit.html.slim (hunk 2 above)
Comment (KamilFilip) [inline app/views/users/edit.html.slim]: I was thinking if we can add smth there, the page looks rather.. empty :)

**Comment by bpietraga** [comment]
+1

File: app/views/users/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    .grid-item class="4/4"
+      =form_for(current_user) do |edit|
+        =edit.label :name
+        =edit.text_field :name
+        =edit.label :surname
+        =edit.text_field :surname
+        =edit.label :city
+        =edit.text_field :city
+        br
+        br
```
Comment (gotar) [inline app/views/users/edit.html.slim]: :trollface:

**Comment by gotar** [comment]
+1

**Pull request merged**